Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -243,18 +238,10 @@ private IEnumerator ShootPaintball()
             waitingToShoot = true;
            
             GameObject paintball;
-            if (photonView.IsMine)  //We check if this is the local player shooting
-            {
-                paintball = Instantiate(paintballPrefab, paintGun.transform.position, Quaternion.identity);
-                paintball.GetComponent<PaintBall>().playerWhoShot = this.gameObject;
-                paintball.GetComponent<PaintBall>().paintballDamage = this.paintballDamage;
-            }
-            else //we spawn fake bullets for other players
-            {
-                paintball = Instantiate(paintballPrefabClient, paintGun.transform.position, Quaternion.identity);
-                paintball.GetComponent<PaintballClientSide>().playerWhoShot = this.gameObject;
-            }
-            
+            paintball = Instantiate(paintballPrefab, paintGun.transform.position, Quaternion.identity);
+            paintball.GetComponent<PaintBall>().playerWhoShot = this.gameObject;
+            paintball.GetComponent<PaintBall>().paintballDamage = this.paintballDamage;
+            paintball.GetComponent<PaintBall>().isLocal = photonView.IsMine;
```

Comment:
Why do we replace all this logic?

Reply:
We now use a single prefab with a bool instead of 2 seperate prefabs. Makes the whole system cleaner with less copy pasted code.

Reply:
Great!